frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -191,20 +184,8 @@ const DeviceUserPage = ({
       retry: false,
       // TODO: refactor to use non-refetch data directly in the component and remove
       // unnecesary derived states for values that aren't related to the refetch status
```

Comment:
Is this TODONE based on your migrating things like `isPremiumTier` to regular vars rather than state vars?

Reply:
Ha, didn't even notice this comment. @gillespi314 has good taste in refactors.

The second part is TODONE, will take a look at the first part after lunch.

Reply:
Merging to keep things moving – follow-up PR with said refactor on the way

Reply:
Oh, yes it _is_ all now TODONE – removing this comment [here](https://github.com/fleetdm/fleet/pull/25399)